### PR TITLE
Set static page titles to avoid reporting empty page titles in analytics

### PR DESCRIPTION
**Pull request by pavinduLakshan**
Note $subject

### Related Issues
- Issue `#1` or (None)

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/identity-providers/pages/identity-provider-template.tsx [hunk 1]
```diff
@@ -400,6 +400,7 @@ const IdentityProviderTemplateSelectPage: FunctionComponent<IdentityProviderTemp
 
     return (
         <PageLayout
+            pageTitle="Create New Connection"
```
Comment (Rashmini) [inline apps/console/src/features/identity-providers/pages/identity-provider-template.tsx:403]:
```suggestion
            pageTitle={ "Create New Connection" }
```

Comment (pavinduLakshan) [reply apps/console/src/features/identity-providers/pages/identity-provider-template.tsx:403]: Fixed in 8f5df42

Comment (pavinduLakshan) [reply apps/console/src/features/identity-providers/pages/identity-provider-template.tsx:403]: Fixed in 8f5df42

File: apps/console/src/features/identity-providers/pages/identity-provider-edit.tsx [hunk 2]
```diff
@@ -565,6 +565,7 @@ const IdentityProviderEditPage: FunctionComponent<IDPEditPagePropsInterface> = (
 
     return (
         <PageLayout
+            pageTitle={ "Edit Connection" }
```
Comment (NipuniBhagya) [inline apps/console/src/features/identity-providers/pages/identity-provider-edit.tsx:568]: Let's remove the unnecessary curly brackets.

Comment (pavinduLakshan) [reply apps/console/src/features/identity-providers/pages/identity-provider-edit.tsx:568]: Fixed with 509f01e

**Pull request merged**